test/test_timestamps.py

```diff
@@ -101,6 +77,8 @@ def test_increase_time():
 
     # increase time by 1 day
     increase_time(86400)
+    ts_after_increase_time = get_ts_from_last_block()
+    assert ts_after_increase_time >= ts_after_deploy + 86400
```

Comment:
Should we use `==` instead of `>=`?

Reply:
on my machine these timestamps can differ 0-4s, so sometimes in = and sometimes there is a shift so this is why there is `>=`

`>=` have been before in these tests

Reply:
Ok, I thought that instantly mining a block would make `==` pass

Reply:
not so instant on my machine :D